src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line before the class declaration.